src/components/standalone/firewall/conntrack/ConntrackRecordsTable.vue

```diff
@@ -0,0 +1,110 @@
+<!--
+  Copyright (C) 2024 Nethesis S.r.l.
+  SPDX-License-Identifier: GPL-3.0-or-later
+-->
+
+<script setup lang="ts">
+import { useI18n } from 'vue-i18n'
+import {
+  NeButton,
+  NeTable,
+  NeTableHead,
+  NeTableHeadCell,
+  NeTableBody,
+  NeTableRow,
+  NeTableCell,
+  NePaginator,
+  useItemPagination
+} from '@nethesis/vue-components'
+import type { ConntrackRecord } from './ConntrackContent.vue'
+import { ref, defineProps } from 'vue'
+import { byteFormat1024 } from '@nethesis/vue-components'
+const { t } = useI18n()
+
+const props = defineProps<{
+  conntrackRecords: ConntrackRecord[]
+  loading?: boolean
+}>()
+
+const emit = defineEmits(['record-delete'])
```

Comment:
Since an object is passed on with an event, please change the `defineEmits` using the typescript version
```suggestion
const emit = defineEmits<{
  recordDelete: [record: ConntrackRecord]
}>()
```

(this should be the correct code, don't quote me on that)